src/components/ErrorMessage/styles.js

```diff
@@ -0,0 +1,7 @@
+import styled from "styled-components";
+
+export const Errortext = styled.p`
```

Comment:
In React we use PascalCase, so it should be `ErrorText`
```suggestion
export const ErrorText = styled.p`
```